rebasebot/lifecycle_hooks.py

```
@@ -67,49 +70,50 @@ def __init__(self, script_location: str):
 
-    def _fetch_from_local_git(
-            self, gitwd: git.Repo, git_ref: str, file_path: str, script_file_path: str):
+    def _fetch_from_local_git(self, gitwd: git.Repo, git_ref: str, file_path: str, script_file_path: str):
         """Fetches script from local git repository."""
         git_path = f"{git_ref}:{file_path}"
         try:
             script_content = _retrieve_file_from_git(gitwd, git_path)
-            with open(script_file_path, "w", encoding='latin1') as f:
+            with open(script_file_path, "w", encoding="latin1") as f:
                 f.write(script_content)
                 os.chmod(script_file_path, 0o755)  # Make it executable
```

Ruff (0.15.10)

[error] `os.chmod` setting a permissive mask `0o755` on file or directory

(S103)